ccm-connector/src/main/java/com/sequenceiq/cloudbreak/ccmimpl/cloudinit/DefaultCcmV2JumpgateParameterSupplier.java

```diff
@@ -29,10 +29,10 @@ public CcmV2JumpgateParameters getCcmV2JumpgateParameters(@Nonnull String accoun
 
         LOGGER.debug("CcmV2JumpgateConfig successfully retrieved InvertingProxyHost: '{}', InvertingProxyStatus: '{}', InvertingProxyAgentCrn: '{}', " +
                         "EnvironmentCrnOpt: '{}'", invertingProxy.getHostname(), invertingProxy.getStatus(),
```

Comment:
`EnvironmentCrnOpt`: Technically this is not an `Optional` anymore, so I suggest renaming it to `EnvironmentCrn`. I presume it may not be `null`, otherwise the `Optional.of()` call would have failed earlier.